packages/cloud_functions/cloud_functions_platform_interface/lib/src/method_channel_cloud_functions.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of cloud_functions_platform_interface;
+
+/// Exception that can be thrown by [MethodChannelCloudFunctions] to report
+/// errors that occurred inside the channel. A convenience wrapper for [PlatformException].
+class CloudFunctionsException implements Exception {
+  CloudFunctionsException._(this.code, this.message, this.details);
+
+  /// Error code reported by the platform
+  final String code;
+
+  /// Error message reported by the platform
+  final String message;
+
+  /// Additional info provided by the platform's exception
+  final dynamic details;
+}
+
+/// [CloudFunctionsPlatform] implementation that delegates to a [MethodChannel].
+class MethodChannelCloudFunctions extends CloudFunctionsPlatform {
+  /// The [MethodChannel] to which calls will be delegated.
+  @visibleForTesting
+  static const MethodChannel channel = MethodChannel(
+    'plugins.flutter.io/cloud_functions',
+  );
+
+  @override
+  dynamic callCloudFunction({
+    @required String appName,
+    @required String functionName,
+    String region,
+    String origin,
+    Duration timeout,
+    dynamic parameters,
+  }) async {
+    try {
+      final dynamic response = await channel
+          .invokeMethod<dynamic>('CloudFunctions#call', <String, dynamic>{
+        'app': appName,
+        'region': region,
+        'origin': origin,
+        'timeoutMicroseconds': timeout?.inMicroseconds,
```

Comment:
I think this should be `'timeoutMilliseconds': timeout?.inMilliseconds,` based on https://github.com/FirebaseExtended/flutterfire/blob/master/packages/cloud_functions/android/src/main/java/io/flutter/plugins/firebase/cloudfunctions/CloudFunctionsPlugin.java#L52

Reply:
Does that mean that the existing implementation is a bug?
https://github.com/FirebaseExtended/flutterfire/blob/master/packages/cloud_functions/lib/src/https_callable.dart#L38

I mean, I'm happy to make the change if it's the right thing to do, but I don't want to break something that works, currently.

Reply:
It seems that the Android code is the odd one out. iOS is also using microseconds:

`NSNumber *timeoutMicroseconds = call.arguments[@"timeoutMicroseconds"];`

https://github.com/FirebaseExtended/flutterfire/blob/master/packages/cloud_functions/ios/Classes/CloudFunctionsPlugin.m#L48

Reply:
Yes, it looks like a bug in the Android implementation